lib/srv/monitor.go

```diff
@@ -223,8 +223,9 @@ func (w *Monitor) start(lockWatch types.Watcher) {
 				lock, ok := lockEvent.Resource.(types.Lock)
 				if !ok {
 					w.Entry.Warnf("Skipping unexpected lock event resource type %T.", lockEvent.Resource)
+				} else {
```

Comment:
I *think* this was a minor bug, as we didn't actually skip anything - we returned a `LockInForce` error even if the type assertion failed.